usmqe/gluster/gluster.py

```diff
@@ -202,7 +189,39 @@ def info(self):
 
         Returns:
             dictionary: parsed json from
```

Comment:
Does it really return any dictionary?
Please correct the description here. BTW the command is corrected but not everywhere.